pkg/daemon/daemon.go

```diff
@@ -147,9 +149,16 @@ func NewDaemon(cfg *rest.Config, bindAddress string) (Daemon, error) {
 		return nil, fmt.Errorf("failed to new crr daemon controller: %v", err)
 	}
 
+	// node pod probe
+	nppController, err := podprobe.NewController(opts)
```

Comment:
plz add a feature gate for node pod probe, so that we can disable the feature more easily